hooks/snyk-code.sh

```diff
@@ -1,6 +1,25 @@
 #!/usr/bin/env bash
+
 set -eu
```

Comment:
```suggestion
set -u
```
Since you do `set +e` in the very next instruction. Same comment for other scripts.